analyses/selection-strategy-comparison/README.md

```diff
@@ -0,0 +1,15 @@
+## Explore RNA-seq selection strategies
+
+**Module author:** Joshua Shapiro ([@jashapiro](https://github.com/jashapiro))
+
+This analysis was originally undertaken prior to [`release-v5-20190924`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/doc/release-notes.md#release-v5-20190924), where samples from the two library strategies (stranded, poly-A) were split into separate files.
```

Comment:
The wording is slightly ambiguous to me so I've written what I think this means. Feel free to edit the wording itself, just wanted clarification on what lead to what. 
```suggestion
This analysis was originally undertaken prior to [`release-v5-20190924`](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/doc/release-notes.md#release-v5-20190924). 
Data releases prior to v5 had samples from the two library strategies (stranded, poly-A) in the same file, but due to the findings of this analysis, libraries of different strategies were split into separate files in the subsequent data releases. 
```